README.md

```diff
@@ -166,7 +166,7 @@ The currently supported commands are:
 | table       | ✅       | ✅   |
 | tree        | ✅       | ❌   |
 | calendar    | ✅       | ❌   |
-| padder      | ✅       | ❌   |
+| padder      | ✅       | ✅   |
```

Comment:
Since according to this (and according to my opinion :)) the padder module seems to be complete now, the `progress.fs` and the according image should be updated, too

Reply:
Correct. Done.